dis_snek/models/discord/guild.py

```diff
@@ -1230,13 +1230,19 @@ async def fetch_channel(self, channel_id: Snowflake_Type) -> Optional["models.TY
 
         """
         channel_id = to_snowflake(channel_id)
-        if channel_id in self._channel_ids:
+        if channel_id in self._channel_ids or self._client.start_time is MISSING:
```

Comment:
I would rather add a property to client (Snake) like `client.gateway_started` and use this property here in such checks

Reply:
Resolved in my latest commit